monteCarloTreeSearch.c

```diff
@@ -0,0 +1,383 @@
+/*
+* Author: Jenna Bennett
+* Date: March 30, 2017
+* File: monteCarloTreeSearch.c
+*
+* Description: This file contains the code for the main of Othello and the monte Carlo Tree Search
+* code that will run the game Othello and determine the best move to be made.
+* Compile using:
+* gcc -lm -Wall -o monteCarloTreeSearch monteCarloTreeSearch.c board.c node.c
+
+*/
+
+#include <time.h>
+#include <stdlib.h>
+#include <stdio.h> 
+#include <math.h>
+#include "node.h"
+#include "board.h"
+
+
+void pickMove(struct board* gameBoard, int nodeLimit, int timeLimit, int* move)
+{
+	node root; 				//root node
+	node parentNode;           //node to hold the parentNode for backPropagation
+	node newNode;              // tempNode for looping
+	node selected;				//selected value for expansion
+	node child;				//child node pointer
```

Comment:
Actually, I take it back. You should make all of these pointers and just malloc() the root node. That way you can just call deconstructTree() on root without having to worry about accidentally calling free() on it.